tests/lib/token-store.js

```diff
@@ -89,6 +90,38 @@ describe("TokenStore", () => {
             );
         });
 
+        it("should retrieve all matched tokens for root node with filter", () => {
+            check(
+                store.getTokens(Program, t => t.type === "Identifier"),
+                ["answer", "a", "b", "call"]
+            );
+            check(
+                store.getTokens(Program, { filter: t => t.type === "Identifier" }),
+                ["answer", "a", "b", "call"]
+            );
+        });
+
+        it("should retrieve all tokens and comments in the node for root node with includeComments option", () => {
+            check(
+                store.getTokens(Program, { includeComments: true }),
+                ["var", "answer", "B", "=", "C", "a", "D", "*", "b", "E", "F", "call", "(", ")", ";"]
+            );
+        });
+
+        it("should retrieve matched tokens and comments in the node for root node with includeComments and filter option", () => {
+            check(
+                store.getTokens(Program, { includeComments: true, filter: t => t.type === "Block" }),
```

Comment:
For the time being, I modified it with `filter: t => t.type.startsWith("Block")`